pkg/storaged/resize.jsx

```diff
@@ -245,19 +247,74 @@ export function free_space_after_part(client, part) {
 
 export function grow_dialog(client, lvol_or_part, info, to_fit) {
     let title, block, name, orig_size, max_size, allow_infinite, round_size;
+    let has_subvols, subvols, pvs_as_spaces, initial_pvs;
+
+    function compute_max_size(spaces) {
+        const layout = lvol_or_part.Layout;
+        const pvs = spaces.map(s => s.pvol);
+        const n_pvs = pvs.length;
+        const sum = pvs.reduce((sum, pv) => sum + pv.FreeSize, 0);
+        const min = Math.min.apply(null, pvs.map(pv => pv.FreeSize));
+
+        if (!has_subvols) {
+            return sum;
+        } else if (layout == "raid0") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20231026-132112-61c336d9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)